lib/index.js

```diff
@@ -53,13 +53,11 @@ Drip.prototype.loaded = function() {
  */
 
 Drip.prototype.track = function(track) {
-  var props = {};
-  each(function(value, key) {
-    var formattedKey = key.replace(' ', '_');
-    props[formattedKey] = value;
-  }, track.properties());
+  var props = format(track.properties());
```

Comment:
format is a little general of a name for a function that has such a specific purpose, if it were me I'd name it `replaceSpaces` or something like that but that's just my opinion

Reply:
hahah I actually was going to do that but I made it a bit broader in case we need to ship additional translation/formatting logic in the future but yes! I generally always prefer more accurate/explicit function names :)